internal/sdkprovider/service/account/account_team_member.go

```diff
@@ -205,21 +205,15 @@ func resourceAccountTeamMemberDelete(ctx context.Context, d *schema.ResourceData
 	}
 
 	// delete account team member
-	found := false
 	for _, m := range r.Members {
 		if m.UserEmail == userEmail {
 			err = client.AccountTeamMembers.Delete(ctx, accountID, teamID, m.UserId)
 			if err != nil && !aiven.IsNotFound(err) {
 				return diag.FromErr(err)
 			}
-			found = true
 			break
 		}
 	}
 
-	if !found {
-		return diag.Errorf("user with email %q is not a part of the team %q", userEmail, teamID)
-	}
-
```

Comment:
Please leave a comment to explain why we do this.

Reply:
done!